frontend/packages/kubevirt-plugin/src/components/vms/table-filters.ts

```diff
@@ -1,18 +1,18 @@
 import * as _ from 'lodash';
 import { Filter } from '@console/shared';
-import { VM_SIMPLE_STATUS_ALL, VM_SIMPLE_STATUS_TO_TEXT } from '../../statuses/vm/constants';
-import { getSimpleVMStatus } from '../../statuses/vm/vm';
+
+const statuseStrings = ['Starting', 'Migrating', 'Stopping', 'Running', 'Off'];
```

Comment:
can you move it to `statuses/vm/constants` so these constant stay together? Though it would be much nicer to refactor it all to enums.

Reply:
moved.